src/reactions/reactionsSystems/MixedEquilibriumKineticReactions_impl.hpp

```diff
@@ -172,23 +168,20 @@ MixedEquilibriumKineticReactions< REAL_TYPE,
   {
     for( IntType j = 0; j < numPrimarySpecies; ++j )
     {
-      dReactionRates_dLogPrimarySpeciesConcentrations( i, j ) = reactionRatesDerivatives( i, j + numSecondarySpecies );
+      dReactionRates_dLogPrimarySpeciesConcentrations( i, j ) = logmath::dWrtLogConst< REAL_TYPE >() * reactionRatesDerivatives( i, j + numSecondarySpecies );
```

Comment:
I think it is better to account for `logmath::dWrtLogConst< REAL_TYPE >() ` when we calculate `reactionRatesDerivatives` in `kineticReactions::computeReactionRates` instead of here.